flocker/testtools/_base.py

```diff
@@ -86,8 +93,25 @@ def __init__(self, *args, **kwargs):
 
     def setUp(self):
         super(AsyncTestCase, self).setUp()
+        # Need this to run _after_ the clean-up in CaptureTwistedLogs.
+        #
+        # Would ideally like to move post-processing to its own fixture that
+        # wraps up CaptureTwistedLogs, but there doesn't seem to be a way to
+        # do that. https://github.com/testing-cabal/fixtures/pull/20 for
+        # details.
```

Comment:
> Even if this can't be a fixture wrapping another fixture, would it make sense to still abstract it as a fixutre that needs to be added before CaptureTwistedLogs?

I don't think so, actually. Doing so ends up with two very closely intertwined fixtures that appear independent.

> Actually, as I commented here I think it is actually possible to post-process details. You just can't use a cleanup to do it.

Noted. See below.